Assets/MixedRealityToolkit/_Core/Utilities/AutoStartBehavior.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Core.Definitions.Utilities
```

Comment:
The namespace doesn't match the folder. Shouldn't this be in the `Assets/MixedRealityToolkit/_Core/Definitions/Utilities` folder?

Reply:
I didn't realize he moved it 😅